requirements/prod.txt

```diff
@@ -154,9 +163,9 @@ six==1.16.0
     #   python-dateutil
     #   salesforce-bulk
     #   snowfakery
-snowfakery==2.1
+snowfakery==2.2
```

Comment:
This needs to happen along with corresponding changes in #2962 right @prescod?

Reply:
Sorry yes, I got focused on the Windows installer work and it slipped my mind that I need to finish #2962 . I'll switch to that now. @Br4nd0R

Reply:
Per my discussion with @prescod, I'm pinning snowfakery to 2.1 and Paul will undo the pin on a subsequent PR.